.circleci/config.yml

```diff
@@ -499,7 +499,7 @@ aliases:
     service-memory-limit-2: 8Gi
     service-storage-size-2: 500Gi
     service-name-3: indexer
-    service-image-3: registry.gitlab.com/thorchain/midgard:2.13.0
+    service-image-3: registry.gitlab.com/thorchain/midgard:2.15.1
```

Comment:
let's just keep the thornode upgrade. last time I tried to bump midgard, it did not go well